src/algorithms/calorimetry/CalorimeterHitReco.cc

```diff
@@ -44,6 +44,14 @@ void CalorimeterHitReco::init(const dd4hep::Detector* detector, const dd4hep::re
     m_log = logger;
 
     // threshold for firing
+    // Should set either m_cfg.thresholdFactor or m_cfg.thresholdValue, not both
+    if ( m_cfg.thresholdFactor > 0 &&  m_cfg.thresholdValue > 0){
+        m_log->error("thresholdFactor = {}, thresholdValue = {}. Only one of these should be non-zero.",
+                    m_cfg.thresholdFactor, m_cfg.thresholdValue);
+        // throw std::exception(fmt::format("CalorimeterHitReco: thresholdFactor = {}, thresholdValue = {}. Only one of these should be non-zero.",
+        //                                   m_cfg.thresholdFactor, m_cfg.thresholdValue));
+        throw;
```

Comment:
Can you throw a std::exception with what arg, https://en.cppreference.com/w/cpp/error/exception? e.g.
```suggestion
        throw std::runtime_error("thresholdFactor and thresholdValue should not both be set");
```

Reply:
I guess I see now that there are some comments to that regard. That commented code should be removed if it isn't going to be activated.

Reply:
Fair, but I think replacing it with a comment `// throw with an argument doesn't trigger abort` or something like that is appropriate (so that I don't get this question again in the future :) )